.github/workflows/build.yml

```diff
@@ -126,7 +126,7 @@ jobs:
         uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          password: ${{ secrets.DOCKERHUB_TOKEN || secrets.DOCKERHUB_PASSWORD }}
```

Comment:
Lines like these could just specify the `secrets.DOCKERHUB_PASSWORD` instead of being an OR condition since the `DOCKERHUB_TOKEN` was only used in the repo before its migration to the hirosystems org.